content/articles/2019-08-gargle-hello-world.Rmarkdown

```diff
@@ -0,0 +1,88 @@
+---
+title: gargle's debut on CRAN
+slug: gargle-hello-world
+author: Jenny Bryan
+description: >
+  gargle is now on CRAN.
+date: '2019-08-21'
+categories: [package]
+tags:
+  - gargle
+photo:
+  url: https://flic.kr/p/oe2LJ1
+  author: North Carolina Christian advocate (1894)
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+## Introduction
+
+We're elated to announce the initial release of gargle and of googledrive v1.0.0 on CRAN!
```

Comment:
```suggestion
We're elated to announce the initial CRAN release of gargle v1.0.0 and googledrive v1.0.0!
```

Reply:
Or should this paragraph just focus on gargle? I think maybe you could now remove the bulleted list, keeping just the paragraph encouraging people to also read about googledrive.

Reply:
I'll rework this intro.